addons/event_sale/models/event_ticket.py

```diff
@@ -1,139 +1,10 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-
-import logging
-
-from odoo import api, fields, models
-from odoo.addons.product.models.product_template import PRICE_CONTEXT_KEYS
-
-_logger = logging.getLogger(__name__)
-
-
-class EventTemplateTicket(models.Model):
```

Comment:
While moving this code, you can also move test setup / code that goes with it... notably ``TestEventSaleCommon`` that contains products and tickets, I suppose it can be moved in event_product. Then tests like ``test_event_configuration_from_type`` and ``test_event_change_max_seat_no_side_effect`` can be moved I think. To check a bit more in details, but that way we lower tests at first place they are really usefull :)